src/Login/LoginStyles/login.scss

```diff
@@ -0,0 +1,107 @@
+
+@import '../../commons/assets/variables/font.scss';
+@import '../../commons/assets/variables/colors.scss';
+
+.Login-container{
+  margin: 10px 0 20px 0;
+
+  .myform{
+  width: 517px;
+  height: 480px;
+  border: #cccccc solid 1px;
+  margin-top: 30px;
+  margin-left: auto;
+  margin-right: auto;
+  padding-top: 70px;
+  }
+
+  .heading {
+    width: 387px;
+    height: 30px;
+    margin-left: auto;
+    margin-right: auto;
+    text-align: center;
+    opacity: 99%;
+  }
+  .form-input input, button{
+    margin-top: 30px;
+    width: 297px;
+    margin-left: 99px;
+    height: 40px;
+    padding: 6px 6px 6px 10px;
+    font-size: 14px;
+
+  }
+  .mypara{
+    color:#606060;
+    text-align: left;
+    font-family: 'Source Sans Pro';
+    margin-left: 100px;
+    margin-top: 20px;
+  }
+
+  ::placeholder{
+    font-family: 'Source Sans Pro';
+    font-size: 18px;
+    font-weight: Regular;
+  }
+  button{
+    color: #eaeff0;
+    background-color: #00a4d3;
+    font-size: 18px;
+    margin-top: 0px;
+    height: 38px;
+    margin-bottom: 10px;
+    font-weight: 100;
+   font-family: $lato-font;
+  }
+  .fa:hover {
+    opacity: 0.7;
+  }  
+  .mypara2{
+    color: #00a4d3;
```

Comment:
@jennizalwango kindly add font-weight:100; here also to have the right font.

Reply:
done
@alexxsanya tahns and
 kindly note it is done